test/protocol/TwinHandlerTest.js

```diff
@@ -557,6 +555,7 @@ describe("IBosonTwinHandler", function () {
           await twinHandler.connect(assistant).createTwin(twin);
 
           // Create new twin with same token address
+          // twin.supplyAvailable = "2"
```

Comment:
👀

Reply:
I guess I was testing my findings that led to #596 and I forgot to uncomment this.
I think it shouldn't affect the result, so I uncommented it.